packages/mm-backend/src/common/utils/auth/jwt.strategy.spec.ts

```diff
@@ -0,0 +1,35 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { JwtStrategy } from './jwt.strategy';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+
+describe('JwtStrategy', () => {
+  let strategy: JwtStrategy;
+
+  beforeEach(async () => {
+    const module: TestingModule = await Test.createTestingModule({
+      imports: [
+        ConfigModule.forRoot({
```

Comment:
Add a test to check if a user can access only his orders, strategies etc

Reply:
In 308b9ee5 access_token now includes clientId (mixin user id) which will be used to fetch user resources